packages/table/src/locator.ts

```diff
@@ -24,24 +24,22 @@ export interface ILocator {
     getTallestVisibleCellInColumn: (columnIndex: number) => number;
 
     /**
-     * Locates a column's index given the client X coordinate. Returns -1 if
-     * the coordinate is not over a column.
+     * Locates a column's index given the client X coordinate.
      * If `useMidpoint` is `true`, returns the index of the column whose left
      * edge is closest, splitting on the midpoint of each column.
      */
     convertPointToColumn: (clientX: number, useMidpoint?: boolean) => number;
 
     /**
-     * Locates a row's index given the client Y coordinate. Returns -1 if
-     * the coordinate is not over a row.
+     * Locates a row's index given the client Y coordinate.
      * If `useMidpoint` is `true`, returns the index of the row whose top
      * edge is closest, splitting on the midpoint of each row.
      */
     convertPointToRow: (clientY: number, useMidpoint?: boolean) => number;
 
     /**
      * Locates a cell's row and column index given the client X
-     * coordinate. Returns -1 if the coordinate is not over a table cell.
```

Comment:
If you _don't_ return -1 from here—and instead return the next index beyond the index currently in view—then the viewport will move to out-of-view indices as desired. If you return -1, the viewport doesn't update, making the whole drag-to-scroll feature stop working.